FirebaseAuth/Sources/Backend/FIRAuthBackend.m

```diff
@@ -641,7 +645,22 @@ + (NSMutableURLRequest *)requestWithURL:(NSURL *)URL
   if (languageCode.length) {
     [request setValue:languageCode forHTTPHeaderField:kFirebaseLocalHeader];
   }
-  return request;
+  if (requestConfiguration.appcheck) {
+    [requestConfiguration.appcheck
+        getTokenForcingRefresh:false
+                    completion:^(id<FIRAppCheckTokenResultInterop> _Nonnull tokenResult) {
+                      if (tokenResult.error) {
+                        FIRLogError(kFIRLoggerAuth, @"I-AUT000001",
```

Comment:
This error message and code needs to be updated (and also changed to `FIRLogWarning`).

Reply:
Have updated the code to a unique number,  where can I find the correct message?

Reply:
Looks like the code update was accidentally made in the wrong location! (I added a comment there as well)

For the error message, you can use: `@"Error getting App Check token; using placeholder token instead. Error: %@"`